packages/oasis-actions/src/types/ajna/AjnaPosition.ts

```diff
@@ -94,6 +96,13 @@ export class AjnaPosition implements IAjnaPosition {
     return new RiskRatio(normalizeValue(loanToValue), RiskRatio.TYPE.LTV)
   }
 
+  originationFee(quoteAmount: BigNumber) {
+    return getAjnaBorrowOriginationFee({
+      interestRate: this.pool.interestRate,
+      quoteAmount,
+    })
+  }
+
```

Comment:
Can you move origininationFee to simulation? 

I would add new fields to simulation called fees which will be an array of fees, with amount, token and maybe name 

What do you think?

Reply:
I'm using this method in borrow as well so it must be defined on constructor level.